src/App.tsx

```diff
@@ -15,47 +15,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [sortType, setValue] = useState(SortType.NONE);
```

Comment:
```suggestion
  const [sortType, setSortType] = useState(SortType.NONE);
```

Reply:
in react for state we are using naming style
`[{state field}, set{state field}]`